internal/reports/project.go

```diff
@@ -68,12 +68,12 @@ var (
 // reports with the highest detail levels can be several MB large, we don't just
 // return them all in a big list. Instead, the `submit` callback gets called
 // once for each project report once that report is complete.
-func GetProjectResources(cluster *core.Cluster, domain db.Domain, project *db.Project, now time.Time, dbi db.Interface, filter Filter, submit func(*limesresources.ProjectReport) error) error {
+func GetProjectResources(cluster *core.Cluster, domain db.Domain, project *db.Project, now time.Time, dbi db.Interface, filter Filter, submit func(*limesresources.ProjectReport) error, serviceInfos map[db.ServiceType]liquid.ServiceInfo) error {
```

Comment:
On a function that takes a callback, it's usually good to put the callback as the last argument to make the callsite more readable (i.e., to avoid separating the arguments trailing behind the callback from the line containing the start of the function call, if the callback is a multi-line anonymous function).
```suggestion
func GetProjectResources(cluster *core.Cluster, domain db.Domain, project *db.Project, now time.Time, dbi db.Interface, filter Filter, serviceInfos map[db.ServiceType]liquid.ServiceInfo, submit func(*limesresources.ProjectReport) error) error {
```